src/sentry/models/platformexternalissue.py

```diff
@@ -22,3 +22,12 @@ class Meta:
         unique_together = (('group_id', 'service_type'), )
 
     __repr__ = sane_repr('group_id', 'service_type', 'display_name', 'web_url')
+
+    @classmethod
+    def get_annotations(cls, group):
+        external_issues = cls.objects.filter(group_id=group.id)
+        annotations = []
+        for ei in external_issues:
+            annotations.append('<a href="%s">%s</a>' % (ei.web_url, ei.display_name))
```

Comment:
Do we know that display_name won't contain any HTML data?

Reply:
this is where we set it: https://github.com/getsentry/sentry/blob/master/src/sentry/mediators/external_issues/issue_link_creator.py#L39

but also I'm going to see how hard it would be to convert these all to be rendering in the UI w/o injecting html (per your comment a thousand years ago)

Reply:
From that link it looks like the 3rd part can send whatever they'd like back in their response and we'll add it into the DOM.